app/views/editLiability/disposeLiabilityHasBankDetails.scala.html

```diff
@@ -51,8 +51,7 @@
         "true" -> messages("ated.label.yes"),
         "false" -> messages("ated.label.no")
         ),
-        '_legend -> messages("ated.bank-details.has-bank-details.header"),
-        '_legendClass -> "visuallyhidden",
+        '_legend -> messages(""),
```

Comment:
This line can be completely removed rather than set empty.

Reply:
Same as the other radio button.

Reply:
Same comment as above